experimental/framework/last-edited/README.md

```diff
@@ -26,13 +35,13 @@ The update should always be called in response to a remote op because:
 
 The details returned in getLastEditDetails contain the `IUser` object and the `timestamp` of the last edit.
 
-# Last Edited Tracker Data Store
+## Last Edited Tracker Data Store
```

Comment:
Fixed inconsistent heading levels. Everything should be parented under the same top-level heading.